<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummarizeDialog</class>
 <widget class="QDialog" name="SummarizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Summarize</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="ql_summarizers">
       <property name="text">
        <string>Summarizers</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="qcbox_summarizers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qpb_options">
         <property name="text">
          <string>Options</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="qcb_includeBuffers">
       <property name="text">
        <string>Include buffers</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="qcb_abbreviateBuffers">
       <property name="text">
        <string>Abbreviate buffers</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="qcb_includeEstimates">
       <property name="text">
        <string>Include estimates</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="qcb_includeIonicEffects">
       <property name="text">
        <string>Include ionic effects</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="qcb_includeCurve">
       <property name="text">
        <string>Include curve</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="ql_title">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="qle_title"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="ql_outputFile">
       <property name="text">
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="qle_outputFile"/>
       </item>
       <item>
        <widget class="QPushButton" name="qpb_browse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="qcb_includeBufferCapacity">
       <property name="text">
        <string>Include buffer capacity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="qcb_includeIonicStrength">
       <property name="text">
        <string>Include ionic strength</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
